lib/SparqlEndpointFetcher.ts

```diff
@@ -38,6 +40,11 @@ export class SparqlEndpointFetcher {
     this.additionalUrlParams = args?.additionalUrlParams ?? new URLSearchParams();
     this.defaultHeaders = args?.defaultHeaders ?? new Headers();
     this.fetchCb = args?.fetch;
+    this.sparqlQueryParser = new SparqlParser({
+      lexerConfig: {
+        positionTracking: 'onlyOffset',
+      },
```

Comment:
Am I correct to assume that this option will be the most commonly used one for people using your parser?
If so, does it make sense to configure this as the default for the zero-args constructor?